mne/evoked.py

```diff
@@ -115,6 +117,37 @@ def __init__(self, fname, condition=None, baseline=None, proj=True,
             self.apply_proj()
         self.data = rescale(self.data, self.times, baseline, copy=False)
 
+    @verbose
+    def apply_baseline(self, baseline, verbose=None):
+        """Baseline correct evoked data
+
+        Parameters
+        ----------
+        baseline : tuple of length 2
```

Comment:
Since read_evokeds also has baseline as a parameter, should it be deprecated there too or should we use the `apply_baseline` inside it ?